database/migrations/5_create_webhooks_table.up.sql

```diff
@@ -3,8 +3,8 @@ CREATE TABLE webhooks(
     ,tokenHeader        VARCHAR(255)
     ,token              VARCHAR(255)
     ,createdAt          TIMESTAMP DEFAULT CURRENT_TIMESTAMP
-    ,lastEmitStatus     VARCHAR(255) DEFAULT ""
-    ,lastEmitTimestamp  TIMESTAMP DEFAULT 0
+    ,lastEmitStatus     VARCHAR(255) DEFAULT ''
+    ,lastEmitTimestamp  TIMESTAMP DEFAULT '1970-01-01 00:00:00'
```

Comment:
wouldn't that change make a migration on sqlite fail?

Reply:
Works for sqlite too